lib/scenario_runner.py

```diff
@@ -1911,6 +1926,11 @@ def cleanup(self, continue_measurement=False):
 
         print(TerminalColors.OKBLUE, '-Cleanup gracefully completed', TerminalColors.ENDC)
 
+
+    def set_filename(self, filename):
```

Comment:
Although superflous I like the expliciteness of this

Reply:
By superfluous I meant that you also can access _original_filename directly. but as said: I like the expliciteness of a function here